cli/standalone/binary.rs

```diff
@@ -598,71 +621,81 @@ impl<'a> DenoCompileBinaryWriter<'a> {
         .with_context(|| format!("Including {}", path.display()))?;
     }
     let mut remote_modules_store = RemoteModulesStoreBuilder::default();
-    let mut code_cache_key_hasher = if self.cli_options.code_cache_enabled() {
-      Some(FastInsecureHasher::new_deno_versioned())
-    } else {
-      None
-    };
+    let mut source_maps = Vec::with_capacity(graph.specifiers_count());
+    // todo(dsherret): transpile in parallel
```

Comment:
Open an issue about it?

Reply:
I'll implement it sometime I'm in the code soon.